org.eclipse.swtchart.examples/src/org/eclipse/swtchart/examples/MultiLevelPieExample.java

```diff
@@ -72,9 +72,10 @@ static public Chart createChart(Composite parent) {
 		// create a chart
 		Chart chart = new Chart(parent, SWT.NONE);
 		// set titles
-		chart.getTitle().setText("Multi-Level Pie Chart");
+		chart.getTitle().setText("Multi Level Pie Chart");
 		// create pie series
-		MultiLevelPie multiLevelPie = (MultiLevelPie)chart.getSeriesSet().createSeries(SeriesType.MULTI_LEVEL_PIE, "countries");
+		// Doughnut multiLevelPie = (Doughnut)chart.getSeriesSet().createSeries(SeriesType.DOUGHNUT, "countries");
+		CircularSeries multiLevelPie = (CircularSeries)chart.getSeriesSet().createSeries(SeriesType.PIE, "countries");
```

Comment:
One may use 
Pie multiLevelPie = (Pie)chart.getSeriesSet().createSeries(SeriesType.PIE, "countries");
here too. The type of the Object multiLevelPie has been kept CircularSeries just to show how one can change the chart from a Pie chart to a Doughnut chart by just changing the enum.